### PR TITLE
Change all links style in Accuracy Checker docs

### PR DESCRIPTION
**Pull request by eaidova**
Previously used in accuracy checker links notation incorrectly works in [online docs](http://docs.openvinotoolkit.org/latest/_tools_accuracy_checker_README.html)

PR https://github.com/opencv/open_model_zoo/pull/251 fixed only several links, I change links for all files.

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/launcher/caffe_installation_readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Install OpenCV 3.3 or later with Python3 bindings
 
-Accuracy Checker uses OpenCV library for image processing. You can miss this step if you are using OpenCV from [OpenVINO toolkit][openvino-get-started].
+Accuracy Checker uses OpenCV library for image processing. You can miss this step if you are using OpenCV from [OpenVINO toolkit](ttps://software.intel.com/en-us/openvino-toolkit/documentation/get-started).
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/launcher/caffe_installation_readme.md]: Typo: ttps

File: tools/accuracy_checker/accuracy_checker/preprocessor/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Accuracy Checker supports following set of preprocessors:
     - `fit_to_window` - adaptive resize keeping aspect ratio for fit image into window with fixed size `[dst_height x dst_width]`,
          but trying to make the image as big as possible.
     - `frcnn_keep_aspect_ratio` - adaptive resize keeping aspect ratio for fit image into window with fixed size `[max_size x max_size]`,
-         but trying to make the minimal dimension of image to be equal to `min_size` or as close to `min_size` as possible, where    
+         but trying to make the minimal dimension of image to be equal to `min_size` or as close to `min_size` as possible, where
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/preprocessor/README.md]: This change shouldn't be in this PR. I would've let it slide, but you didn't even remove _all_ of the trailing spaces. :-)

File: tools/accuracy_checker/accuracy_checker/launcher/caffe_installation_readme.md (hunk 1 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/launcher/caffe_installation_readme.md]: fixed, thanks

File: tools/accuracy_checker/accuracy_checker/preprocessor/README.md (hunk 2 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/preprocessor/README.md]: Now I delete all :) And in other files too

**Pull request merged**